app/src/main/java/com/wednesday/banklocator/model/IfscResponse.kt

```diff
@@ -1,13 +1,26 @@
 package com.wednesday.banklocator.model
 
+
+import androidx.room.Entity
+import androidx.room.Index
+import androidx.room.PrimaryKey
+
+
+@Entity(
+    tableName = "favourite_banks"
+)
 data class IfscResponse(
-    val address: String,
-    val bank: String,
-    val branch: String,
-    val city: String,
-    val contact: String,
-    val district: String,
-    var ifsc: String,
-    val rtgs: Boolean,
-    val state: String
+    @PrimaryKey(autoGenerate = true)
+    var id: Int,
+    var ADDRESS: String,
+    var BANK: String,
+    var BRANCH: String,
+    var CITY: String,
+    var CONTACT: String,
+    var DISTRICT: String,
+    var IFSC: String,
+    var RTGS: Boolean,
+    var STATE: String

```

Comment:
Why are we still keeping this as capital?

Reply:
Actually, These weren't working in small letters.